api/models/education.py

```diff
@@ -1,7 +1,9 @@
 from database import db
 
 class Education(db.Document):
+    classification = db.StringField(required=True)
```

Comment:
Two quick questions, 
- I don't see the `type` property, is this something we need to add to match the data structure on from the `mvc.md` file?
- What this `classification`  property going to hold onto?

Thank you this looks amazing!

Reply:
- `mvc.md` is a brainstorm that we are still updating. Unfortuntaley those updates :)
- `type` is a python native, so we changed it to `classification`. It will hold the type info